config.go

```diff
@@ -344,6 +344,7 @@ func getTLSConfig(cfg *Config) (*tls.Config, *credentials.TransportCredentials,
 	}
 
 	tlsCfg := cert.TLSConfFromCert(certData)
+	tlsCfg.NextProtos = []string{"h2"}
```

Comment:
Are you testing w/ `poold` or `litd`? I'm thinking we might need to also port this over to `lnd` as well to patch it for all clients (as `litd` uses the lnd server as a proxy essentially).

Reply:
Only 'poold', with the exact same python code and libraries I was not able to reproduce with 'lnd'.

Reply:
Interesting. We've had ALPN issues in the past with node.js clients, but this seems to be something different. If you have a bit of time to investigate, could you confirm my hunch that this has something to do with the version of the `google.golang.org/grpc` library? It should be as simple as bumping `google.golang.org/grpc` to `v1.29.1` in `go.mod` of `lnd` and check if you run into the same problem with the exact python code and libraries you used for Pool.

Reply:
I thought the same, but even with the version bump I was unable to reproduce on `lnd` 🤔

Reply:
Weird... So it must be another library then. But thanks for trying that! I assume the go version you compiled Pool and lnd with was also the same?

Reply:
Yes, same version of go